### ISSUE TITLE
Permission error when using the CLI concurrently

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma2#1619]
We run Prisma CLI in parallel, it intermittently fails with this error: 

```
Error: Can't write to /Users/runner/runners/2.164.0/work/introspection-engine-output/introspection-engine-output/node_modules/prisma2/ please make sure you install "prisma2" with the right permissions.
```

**Comment by janpio** [comment]
Duplicate of https://github.com/prisma/prisma2/issues/1605
